qml/pages/ChatPage.qml

```diff
@@ -604,23 +651,49 @@ Page {
                 anchors.horizontalCenter: parent.horizontalCenter
                 spacing: Theme.paddingMedium
 
-                ProfileThumbnail {
-                    id: chatPictureThumbnail
-                    replacementStringHint: chatNameText.text
+                Item {
                     width: chatOverviewItem.height
                     height: chatOverviewItem.height
                     anchors.bottom: parent.bottom
                     anchors.bottomMargin: chatPage.isPortrait ? Theme.paddingMedium : Theme.paddingSmall
 
-                    // Setting it directly may cause an stale state for the thumbnail in case the chat page
-                    // was previously loaded with a picture and now it doesn't have one. Instead setting it
-                    // when the ChatModel indicates a change. This also avoids flickering when the page is loaded...
-                    Connections {
-                        target: chatModel
-                        onSmallPhotoChanged: {
-                            chatPictureThumbnail.photoData = chatModel.smallPhoto;
+                    ProfileThumbnail {
+                        id: chatPictureThumbnail
+                        replacementStringHint: chatNameText.text
+                        width: parent.height
+                        height: parent.height
+
+                        // Setting it directly may cause an stale state for the thumbnail in case the chat page
+                        // was previously loaded with a picture and now it doesn't have one. Instead setting it
+                        // when the ChatModel indicates a change. This also avoids flickering when the page is loaded...
```

Comment:
Is it still the case or could `chatModel.smallPhoto` be bound directly to `chatPictureThumbnail.photoData` these days?

Reply:
Don't know actually. With your recent change, the initialization is fine now, but I simply kept the onSmallPhotoChanged as I haven't had time to test all edge cases again...